src/models/FeedbackModel.ts

```diff
@@ -3,7 +3,7 @@ import { Column, Entity, ManyToOne, PrimaryGeneratedColumn } from 'typeorm';
 import { Feedback, Uuid } from '../types';
 import { UserModel } from './UserModel';
 
-@Entity()
+@Entity('Feedback')
```

Comment:
out of curiosity what happens if you have a string in here vs if you don't?

Reply:
the string specifies what the table is called. If no string it would default to the class name, in this case FeedbackModel.